Automattic/wp-calypso
# Reader full post: remove from section CSS and import style.scss in component instead

Now that we can build CSS assets with Webpack (#26820), this PR switches Reader's full post styles to be loaded by the component rather than via the `reader-full-post` section.

#### Testing instructions

Load some Reader full posts and ensure that the styles are applied correctly. A good test post is:

http://calypso.localhost:3000/read/feeds/58215415/posts/1926776088
